### DISCUSSION TITLE
unable to complete your signup on vercel

### DISCUSSION THREAD

**Discussion by tranhongson2943** [nuxt/nuxt#22654] [Questions (Nuxt 2)]
![Screenshot 2023-08-16 131452](https://github.com/nuxt/nuxt/assets/141892504/d0e2f9b2-7ad1-45c9-8de6-36b7076b5ced)

**Comment by TheAlexLichter** [comment]
This looks more like a problem with Vercel. I don't think we can do much about that here ☺️
